vars/withHdslPod.groovy

```diff
@@ -0,0 +1,57 @@
+import org.centos.contra.Infra.Utils
+
+/**
+ *
+ * @param config
+ * @param body
+ * @return
+ */
+def call(Map config=[:], Closure body){
+
+    infraUtils = new Utils()
+
+    env.podName = config.podName ?: "rhprod-${UUID.randomUUID()}"
+    String openshiftServiceAccount = config.openshift_service_account ?: 'jenkins'
+    String openshiftNamespace = infraUtils.getOpenshiftNamespace()
+    String dockerRegistryURL = infraUtils.getOpenshiftDockerRegistryURL()
+    String ansibleExecutorTag = config.linchpin_executor_tag ?: 'stable'
+    String linchpinExecutorTag = config.linchpin_executor_tag ?: 'stable'
+    String jenkinsContraSlaveTag = config.jenkins_slave_tag ?: 'stable'
+    String ansibleExecutorContainerName = config.productization_container_name ?: 'ansible-executor'
```

Comment:
Update to
`String ansibleContainerName = config.ansible_container_name ?: 'ansible-executor`